docs/rules/prevent-abbreviations.md

```diff
@@ -0,0 +1,81 @@
+# Prevent abbreviations
+
+Using complete words results in more readable code. Not everyone knows all your abbreviations. You only write code once, but it's read many times.
+
+Default replacements are available [here](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/rules/prevent-abbreviations.js#L7).
+
+This rule is fixable for abbreviations with exactly one replacement defined.
+
+
+## Fail
+
+```js
+const e = new Error();
+```
+
+```js
+const e = document.createEvent('Event');
+```
+
+
+## Pass
+
+```js
+const error = new Error();
+```
+
+```js
+const event = document.createEvent('Event');
+```
+
+
+## Options
+
+### replacements
+
+You can extend default replacements by passing this option.
+
+The example below disables the default `e` => `event` replacement and adds a custom `cmd` => `command` one.
+
+```js
+"unicorn/prevent-abbreviations": [
+	"error",
+	{
+		"replacements": {
+			"cmd": {
+				"command": true
+			},
+			"e": {
+				"event": false
+			}
+		}
+	}
+]
+```
```

Comment:
@futpib What do you think about starting to use JS config syntax for our examples? The JSON looks so ugly, and most people use JS config anyway.

Reply:
Ok, but can we defer this to a separate issue too? This would require editing other rules docs.

Reply:
I use `xo` key in `package.json` for configuration though, so it's in json.